### ISSUE TITLE
[RAI] Regular NPCs and Heat Caps

### ISSUE THREAD

**Issue by lenaleciel** [AshleyMoni/Lancer-Community-Edition#372]
**The Problem:**
Currently, the heat cap related text for regular NPCs reads:
"By default, NPCs have 1 reactor stress and gain the exposed condition when they take heat that would put them over capacity."
There has recently been confusion as to what exactly happens to the heat value of the npc afterwards. Does gaining the exposed condition happen alongside gaining heat that would put them over capacity or does it replace them gaining heat that would put them over capacity? 
In other words, does the NPC's heat stay at max if they would take any more heat, or can they take further heat past their heat cap that doesn't have any direct effect? (The difference is mostly relevant for Last Argument of Kings.)

**The Solution:**
Add a line clarifying which result is intended by the developers.

**The Location:**
pg. 364
